nbs/docs/tutorials/ConformalPrediction.ipynb

```diff
@@ -0,0 +1,14030 @@
+{
```

Comment:
<blockquote>these are uncalibrated, meaning that <strong>they tend to ignore seasonality. </strong></blockquote><p>I think we need a more accurate description of what an uncalibrated interval is. Here's a suggestion:</p><blockquote><em>meaning that </em>the actual frequency of observations falling within the interval does not align with the confidence level associated with it. For example, a calibrated 95% prediction interval should contain the true value 95% of the time in repeated sampling. An uncalibrated 95% prediction interval, on the other hand, might contain the true value only 80% of the time, or perhaps 99% of the time. In the first case, the interval is too narrow and underestimates the uncertainty, while in the second case, it is too wide and overestimates the uncertainty.</blockquote><p>Next </p><blockquote><em>Statistical methods also assume normality. Here, we talk about another method called conformal prediction</em> that doesn't require any distributional assumptions. <em>More information... </em></blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/597/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>